src/main/java/com/example/automator/AutomatorController.java

```diff
@@ -120,30 +142,23 @@ public Parameters uploadCSV(@RequestParam("file") MultipartFile file) {
         } catch (IOException e) {
             System.out.println(e);
         }
-
-        //get csv from folder and parse it as Parameters object
-        String CSV_FILE_PATH_DATA = "CSV-files-go-here/data-processed.csv";
-        Parameters parameters = new CSVReader().parseDataCSV(CSV_FILE_PATH_DATA);
-
-        // return parameters from csv
-        return parameters;
     }
 
 
     //Helper methods:
```

Comment:
vielleicht alle helper methods in eine neue Klasse packen? Dann sind im Controller wirklich nur die Endpoints